include/rosdiscover-clang/Value/Float.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <string>
+
+#include "Value.h"
+
+namespace rosdiscover {
+
+class FloatingLiteral : public virtual SymbolicFloat {
+public:
+  FloatingLiteral(double const &literal) : literal(literal) {}
```

Comment:
`double` is probably fine. `APFloat` is even better.

Reply:
sticking to double since APFloat has weird behavior when it models a float (not double)